crates/sui-indexer-alt/src/args.rs

```diff
@@ -24,28 +24,8 @@ pub enum Command {
         #[command(flatten)]
         indexer: IndexerConfig,
 
-        /// How often to check whether write-ahead logs related to the consistent range can be
-        /// pruned.
-        #[arg(
-            long,
-            default_value = "300",
-            value_name = "SECONDS",
-            value_parser = |s: &str| s.parse().map(Duration::from_secs),
-        )]
-        consistent_pruning_interval: Duration,
-
-        /// How long to wait before honouring reader low watermarks.
-        #[arg(
-            long,
-            default_value = "120",
-            value_name = "SECONDS",
-            value_parser = |s: &str| s.parse().map(Duration::from_secs),
-        )]
-        pruner_delay: Duration,
-
-        /// Number of checkpoints to delay indexing summary tables for.
-        #[clap(long)]
-        consistent_range: Option<u64>,
+        #[command(flatten)]
+        sequential_pipeline_config: SequentialPipelineConfig,
```

Comment:
nit: this is even more specific than for sequential pipelines -- let's call it a `ConsistencyConfig` or something?